Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -5877,12 +5877,19 @@ header: <WebCore/PhotoCapabilities.h>
     std::optional<Vector<WebCore::FillLightMode>> fillLightMode;
 };
 
+header: <WebCore/PhotoSettings.h>
+[CustomHeader] struct WebCore::PhotoSettings {
+    std::optional<WebCore::FillLightMode> fillLightMode;
+    std::optional<double> imageHeight;
+    std::optional<double> imageWidth;
+    std::optional<bool> redEyeReduction;
+};
+
 header: <WebCore/RealtimeMediaSource.h>
 [CustomHeader] struct WebCore::PhotoCapabilitiesOrError {
     std::optional<WebCore::PhotoCapabilities> capabilities;
     String errorMessage;
 };
-#endif
```

Comment:
Why this removal?

Reply:
that's likely the cause of the compilation error for tvOS

Reply:
Fixed